packages/angular-sdk-components/src/lib/_bridge/helpers/sdk-pega-component-map.ts

```diff
@@ -115,6 +115,7 @@ import { OperatorComponent } from '../../_components/designSystemExtension/opera
 import { PulseComponent } from '../../_components/designSystemExtension/pulse/pulse.component';
 import { RichTextEditorComponent } from '../../_components/designSystemExtension/rich-text-editor/rich-text-editor.component';
 import { WssQuickCreateComponent } from '../../_components/designSystemExtension/wss-quick-create/wss-quick-create.component';
+import { LocationComponent } from '../../_components/field/location/location.component';
```

Comment:
Let's move this import statement to fields imports and place it alphabetical order